src/ngx_http_lua_headers_in.c

```diff
@@ -645,6 +645,78 @@ ngx_http_clear_builtin_header(ngx_http_request_t *r,
 }
 
 
+static    ngx_hash_init_t             lua_req_header_hash;
+static    u_char header_string[512];
```

Comment:
Global variable is bad, and what does this mean? "to avoid clobbering such data during failed HUP reload."